net/wallet/Portfile

```diff
@@ -0,0 +1,250 @@
+                                port:p${perl5.major}-dbix-class \
+                                port:p${perl5.major}-digest-md5 \
+                                port:p${perl5.major}-sql-translator \
+                                port:p${perl5.major}-timedate
+        depends_build-append    port:p${perl5.major}-crypt-generatepassword \
+                                port:p${perl5.major}-datetime-format-sqlite
+    }
+
+    # TODO: Tests have a Stanford-specific part, and a NetDB-verifier part,
+    # which need to be disabled.
+    # test.run      yes
+    # test.target   check
+
+    # We have our own destroot process, to install just server bits.
+    destroot {
+        # First, install common files from the distribution
+        xinstall -d ${destroot}${prefix}/share/doc/wallet-server
+        xinstall -m 644 ${worksrcpath}/LICENSE \
+            ${worksrcpath}/NEWS \
+            ${worksrcpath}/README \
+            ${worksrcpath}/TODO \
+            ${destroot}${prefix}/share/doc/wallet-server
+
+        # If installing the server or kdc variants, make common directories
+        # These are also created by wallet-kdc
+        xinstall -d ${destroot}${prefix}/etc/remctl/acl
+        xinstall -d ${destroot}${prefix}/etc/remctl/conf.d
+
+        # Install a set of starter remctl ACLs and configurations
+        # ${destroot}${prefix}/etc/remctl/acl is created above
+        xinstall -m 644 ${worksrcpath}/config/wallet-report.acl \
+            ${destroot}${prefix}/etc/remctl/acl/wallet-report
+
+        # ${destroot}${prefix}/etc/remctl/conf.d is created above
+        xinstall -m 644 ${worksrcpath}/config/wallet \
+            ${destroot}${prefix}/etc/remctl/conf.d/wallet
+
+        # Install the wallet server executables
+        # ${destroot}${prefix}/sbin already exists
+        xinstall -m 755 ${worksrcpath}/server/wallet-admin \
+            ${worksrcpath}/server/wallet-backend \
+            ${worksrcpath}/server/wallet-report \
+            ${destroot}${prefix}/sbin
+
+        # Install the wallet server man pages
+        # ${destroot}${prefix}/share/man/man8 already exists
+        xinstall -m 644 ${worksrcpath}/server/wallet-admin.8 \
+            ${worksrcpath}/server/wallet-backend.8 \
+            ${worksrcpath}/server/wallet-report.8 \
+            ${destroot}${prefix}/share/man/man8
+
+        # Install protocol documentataion
+        xinstall -m 644 {*}[glob ${worksrcpath}/docs/*] \
+        ${destroot}${prefix}/share/doc/wallet-server
+
+        # The Perl components of Wallet server are installed by a
+        # Module::Build installer that is normally run by the Makefile.
+        # PERL_INSTALL_ROOT is used to set the destroot path; we don't
+        # set a prefix because that is already configured into MacPorts
+        # Perl, and so Module::Build uses it automatically.
+        set env(PERL_INSTALL_ROOT) ${destroot}
+        ui_info [exec ${prefix}/bin/perl${perl5.major} ${worksrcpath}/perl/Build install]
```

Comment:
Please replace this with
```
system "env 'PERL_INSTALL_ROOT=${destroot}' '${prefix}/bin/perl${perl5.major}' '${worksrcpath}/perl/Build' 'install'"
```

While `ui_info [exec ...]` works, it bypasses the sandbox that MacPorts transparently uses for processes run by `system` to prevent ports from writing in locations where they shouldn't. I've tested that this works.